perllib/FixMyStreet/Cobrand/BathNES.pm

```diff
@@ -92,6 +92,7 @@ sub default_map_zoom {
     # If we're displaying the map at the user's GPS location we
     # want to start a bit more zoomed in than if they'd entered
     # a postcode/address.
+    return unless $self->{c}; # no c for batch job calling static_map
```

Comment:
This presumably should return 3 to match the old behaviour

Reply:
🤦 thanks!